ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -175,6 +175,7 @@ import '../nuxeo-button-styles.js';
 
           #header {
             box-shadow: 0 1px 0 rgba(0, 0, 0, 0.1);
+            padding-left: 2px;
```

Comment:
nit: please consider using `padding-inline-start` here

Reply:
Thank you Sam 👍 I will use it.